salt/modules/genesis.py

```diff
@@ -393,15 +399,22 @@ def _bootstrap_deb(
     if repo_url is None:
         repo_url = 'http://ftp.debian.org/debian/'
 
+    if not salt.utils.which('debootstrap'):
+        log.error('Required tool debootstrap is not installed.')
+        return False
+
     deb_args = [
         'debootstrap',
         '--foreign',
         '--arch',
-        _cmd_quote(arch),
-        '--include',
-    ] + pkgs + [
-        '--exclude',
-    ] + exclude_pkgs + [
+        _cmd_quote(arch)]
+
+    if pkgs:
+        deb_args += ['--include'] + pkgs
+    if exclude_pkgs:
+        deb_args += ['--exclude'] + exclude_pkgs
```

Comment:
At a first glance I wondered the same thing and I decided it was not necessary but since the relative cost of doing it we probably should opt for the safest path. I'll implement it leaving "exploded" commits for your convenience, I'll squash it later when this PR will hopefully get accepted.